webpack/assets/javascripts/services/ExtendableRegistry/index.js

```diff
@@ -0,0 +1,17 @@
+const slotsRegistry = {};
```

Comment:
Could you rename the filename to `slotsRegistry` so developers would know it's apart of the slot&fill?

Reply:
?